.cspell/custom-dictionary.txt

```diff
@@ -107,59 +107,108 @@ Acked
 acks
```

Comment:
is this list maintained with some tooling, or manual? Which linter consumes this?

Reply:
cspell uses this - it's all manual for now. we probably should do something more formal lol.